scripts/pycodestyle_on_repo.py

```diff
@@ -23,14 +23,17 @@
 import subprocess
 import sys
 
+from run_pylint import get_files_for_linting
+
 
 def main():
     """Run pycodestyle on all Python files in the repository."""
     git_root = subprocess.check_output(
         ['git', 'rev-parse', '--show-toplevel']).strip()
     os.chdir(git_root)
-    python_files = subprocess.check_output(['git', 'ls-files', '*py'])
-    python_files = python_files.strip().split()
+    candidates, _ = get_files_for_linting()
```

Comment:
`get_python_files` chunks the set into "library" vs. "tests" -- it is easier here just to search them myself.